src/js/libcs/OpDrawing.js

```diff
@@ -377,7 +377,9 @@ evaluator.drawconic$1 = function(args, modifs) {
             Conic.matrix = arr;
         }
 
+
     }
+    Conic.matrix = List.normalizeMax(Conic.matrix);
```

Comment:
Currently eval_helper.drawconic relies on the fact that aConic is handed over as an object with property matrix, we could of course change that which would entail some reworking of calls of drawconic.